src/layouts/Auth.tsx

```
@@ -83,53 +78,51 @@ const Auth: React.FC<{ light?: boolean }> = ({ light = false }) => {
+          onOpenChange={(open) => {
+            if (!open) {
+              setModalName(null);
+            }
+          }}
+        >
+          <Dialog.Trigger asChild>
+            <FiledBtn
+              children="로그인"
+              color="#4a95f2"
+              onClick={() => setModalName('modal-login')}
+            />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)